samples/bluetooth/iso_central/src/main.c

```diff
@@ -112,7 +112,7 @@ static void device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 	printk("Device found: %s (RSSI %d)\n", addr_str, rssi);
 
 	/* connect only to devices in close proximity */
-	if (rssi < -50) {
+	if (rssi < -70) {
```

Comment:
Revert back... change is not relevant to the scope of this PR.

```suggestion
	if (rssi < -50) {
```